### PR TITLE
Update npm start to run up api

### PR DESCRIPTION
**Pull request by ronanfitz**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+version: "2"
+
+services:
+
+  github-api:
+    image: databraiddb/github-api:latest
+    ports:
+      - 8002:8002
```
Comment (Gwillison415) [inline docker-compose.yml:8]: isn't it the case that we want to map to 8000 in the container? I noticed similar mapping in sheets-api

Comment (joshwyatt) [reply docker-compose.yml:8]: @Gwillison415 See line 10

File: package.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "databraid-react-starter",
+  "name": "github-spa",
```
Comment (joshwyatt) [inline package.json:2]: @ronanfitz thanks.

File: docker-compose.yml (hunk 1 above)
Comment (joshwyatt) [reply docker-compose.yml:8]: @Gwillison415 and also [.env](https://github.com/databraid-dashboard/github-spa/blob/dev/.env)

**Comment by joshwyatt** [comment]
This looks great @ronanfitz, thanks!

**Pull request merged**